hack/push-image.sh

```diff
@@ -55,7 +55,7 @@ fi
 for IMAGE_TO_PUSH in $TOPUSH; do
 	NAME="${IMAGE_TO_PUSH#Dockerfile.}"
 	set -x
-	podman push "${NAME}:${VERSION}" "${REPO}/origin-${NAME}:${VERSION}"
-	podman push "${NAME}:${VERSION}" "${REPO}/origin-${NAME}:latest"
+#	podman push "/localhost/${NAME}:${VERSION}" "${REPO}/origin-${NAME}:${VERSION}"
```

Comment:
Don't think we need this line anymore right?

Reply:
you're right - done!